rke/cis_config.go

```diff
@@ -0,0 +1,27 @@
+package rke
+
+import (
+	v3 "github.com/rancher/types/apis/management.cattle.io/v3"
+)
+
+func loadCisConfigParams() map[string]v3.CisConfigParams {
+	return map[string]v3.CisConfigParams{
+		"default": {
```

Comment:
Setting default has bitten us in the past, aren't CIS versions tied to k8s versions and shouldn't we just declare those and fail if a version is not covered (so we get notified if a version is added without CIS)? Or are we allowing CIS scans not matching k8s versions to be scanned or do we always want a fallback cause of the possible errors?

Reply:
In case, a user has created a cluster with a newer version of k8s before we have released a CIS update, the "default" CIS version is used.

Reply:
CIS doesn't release a new benchmark for every k8s release, hence we need the default.

Reply:
We discussed offline that there is no 1-to-1 mapping for CIS-k8s and that CIS might lag behind on k8s versions, so to cover k8s versions we set a default.